<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>強化学習デモ1</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="epsCount">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="initButton">
         <property name="text">
          <string>Init</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Alpha</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alphaCombo">
         <item>
          <property name="text">
           <string>0.0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.0</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Gamma</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="gammaCombo">
         <item>
          <property name="text">
           <string>0.0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.2</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.0</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Epsilon</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="epsilonCombo">
         <item>
          <property name="text">
           <string>0.0</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.0001</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.001</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.01</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>0.1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1.0</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>ファイル(&amp;F)</string>
    </property>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Exit">
   <property name="text">
    <string>終了(&amp;X)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
